File: integrasjonstest/src/test/java/no/nav/melosys/integrasjonstest/felles/TestSubjectHandler.java
package no.nav.melosys.integrasjonstest.felles;

import no.nav.melosys.sikkerhet.context.SubjectHandler;

public class TestSubjectHandler extends SubjectHandler {
    @Override
    public String getOidcTokenString() {
        return null;
    }

    @Override
    public String getUserID() {
        return "Z990007";
    }
}

File: sikkerhet/src/test/kotlin/no/nav/melosys/sikkerhet/context/TestSubjectHandler.kt
package no.nav.melosys.sikkerhet.context

class TestSubjectHandler : SubjectHandler() {

    override fun getOidcTokenString(): String? = null

    override fun getUserID(): String = "Z990007"

    override fun getUserName(): String = "Testy test"

    override fun getGroups(): List<String> = emptyList()
}
